hypertrace-trace-enricher/helm/values.yaml

```diff
@@ -65,13 +65,32 @@ deploymentSelectorMatchLabels:
 ###########
 traceEnricherConfig:
   name: hypertrace-trace-enricher-config
-  kafka:
-    streams:
-      config:
-        metricsRecordingLevel: INFO
-        numStreamThreads: 2
-        bootstrapServers: "bootstrap:9092"
-        schemaRegistryUrl: "http://schema-registry-service:8081"
+  # Important kafka streams configurations which are used in config template goes here.
+  kafkaStreamsConfig:
+    bootstrapServers: "bootstrap:9092"
+    schemaRegistryUrl: "http://schema-registry-service:8081"
+    #  Core config
+    numStreamThreads: 2 # default = 1
+    commitIntervalMs: 30000 # default = 30000
```

Comment:
If we are keeping the default from kafka, does it make sense to add an `if` in the chart and thus not requiring us to pass a default? cc @JBAhire

Reply:
That makes the chart more complex by adding many if-else branches.

These are very important handpicked kafka/kafka-streams configs that needs to be tuned as per the application performance/throughput requirements

Reply:
Q: Do you think if we can put it as a commented block if a default is the same and it can be uncommented if some values need to be changed?
```
# default = 30000 override it for Xyz
# commitIntervalMs: 30000
```

Reply:
Yah above will again need if/else block in the template which you mentioned. Got it now, that this was done for keeping the template simple.

Reply:
if/else blocks saves us the need to define the defaults ourselves because it adds noise to a already big set of settings. I don't think we will have complex if { if else } else { if else }, it is just a simple if.